CMakeLists.txt

```diff
@@ -227,14 +219,16 @@ endif()
 
 include(CPack)
 
+
 # Add a target to generate API documentation with Doxygen
 find_package(Doxygen)
 if(DOXYGEN_FOUND)
-	configure_file(${CMAKE_CURRENT_SOURCE_DIR}/doc/Doxyfile.in ${CMAKE_CURRENT_BINARY_DIR}/doc/Doxyfile @ONLY)
+	configure_file(${CMAKE_CURRENT_SOURCE_DIR}/doc/Doxyfile.in
+			${CMAKE_CURRENT_BINARY_DIR}/doc/Doxyfile @ONLY)
 	add_custom_target(doc
 		${DOXYGEN_EXECUTABLE} ${CMAKE_CURRENT_BINARY_DIR}/doc/Doxyfile
 		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/doc
 		COMMENT "Generating API documentation with Doxygen" VERBATIM
 	)
-endif(DOXYGEN_FOUND)
+endif()
```

Comment:
Doing this actually *decreases* readability and we don't have CMake style guidelines either.